scripts/docker-compose.yml

```diff
@@ -0,0 +1,109 @@
+name: ecopal
+
+services:
+  # Cassandra Database container to store the embeddings
+  cassandra2:
+    image: cassandra:latest
+    hostname: cassandra2
+    networks:
+      - backend
+    ports:
+      - "9042:9042"
+    volumes:
+      - cassandra2_data:/var/lib/cassandra/data
+
+  # Postgress Database container used for langflow
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_USER: langflow
+      POSTGRES_PASSWORD: langflow
+      POSTGRES_DB: langflow
+    ports:
+      - "5432:5432"
+    volumes:
+      - langflow_postgres:/var/lib/postgresql/data
```

Comment:
Map one level up. /var/lib/postgresql and not /var/lib/postgresql/data

Reply:
Karthik, I noticed the postgres data persist only if we map to /var/lib/postgresql/data. So I reverted the change I made for this.